src/Evolution/DiscontinuousGalerkin/NormalVectorTags.hpp

```diff
@@ -0,0 +1,41 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <optional>
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "DataStructures/Variables.hpp"
+#include "Domain/Structure/Direction.hpp"
+#include "Domain/Structure/DirectionMap.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// Tags used on the interior faces of the elements
+namespace evolution::dg::Tags::InternalFace {
+/// The magnitude of the unnormalized normal covector to the interface
+struct MagnitudeOfNormal : db::SimpleTag {
+  using type = Scalar<DataVector>;
+};
+
+/// The normal covector to the interface
+template <size_t Dim>
+struct NormalCovector : db::SimpleTag {
+  using type = tnsr::i<DataVector, Dim, Frame::Inertial>;
+};
+
+/// The normal covector and its magnitude for all faces of an element.
```

Comment:
is this all faces or all internal faces?

Reply:
I can change this to `all internal faces`. Like I said, not quite clear yet.

Reply:
yes, please make them consistent